openmeter/ent/schema/entitlement.go

```diff
@@ -31,6 +31,8 @@ func (Entitlement) Fields() []ent.Field {
 		field.String("feature_id").Immutable().SchemaType(map[string]string{
 			dialect.Postgres: "char(26)",
 		}),
+		field.Time("active_from").Optional().Nillable().Immutable(),
```

Comment:
when is `active_from` null? For non metered entitlements?

Reply:
For entitlements that don't use scheduling (including all historical ones)